### PR TITLE
Fixed NuGet packaging dependencies

### PR DESCRIPTION
**Pull request by EranGon**

### CODE REVIEW

File: src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj [hunk 1]
```diff
@@ -40,11 +40,9 @@
     <ProjectReference Include="..\SmartDetectors.Extensions\SmartDetectors.Extensions.csproj">
       <PackageDirectory>lib</PackageDirectory>
     </ProjectReference>
+    <ProjectReference Include="..\SmartDetectorsPackage\SmartDetectorsPackage.nuproj" />
```
Comment (ilaizi) [inline src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj]: Why this change?

Comment (EranGon) [reply src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj]: this lets nuproj know that it should take dependency on the SDK's package and not project
this way the SDK's DLL is not dropped to the nupkg of the extensions

---
In reply to: [244003117](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/40#discussion_r244003117) [](ancestors = 244003117)
